### PR TITLE
TOOLS-2556 API documentation should reflect new ownership

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

**Review by nwilkens — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -6,14 +6,15 @@
 
 <!--
     Copyright 2019 Joyent, Inc.
+    Copyright 2023 MNX Cloud, Inc.
 -->
 
 # sdc-cloudapi
 
 This repository is part of the Joyent Triton project. See the [contribution
```
Comment (travispaul) [inline README.md:14]: Missed a reference here.

File: README.md [hunk 2]
```diff
@@ -149,7 +150,7 @@ need to hack the setup the same way we do for testing:
 For more information on RBAC you can check [CloudAPI docs][cloudapi] and
 the [Access Control User Guide][acuguide].
 
-[cloudapi]: https://apidocs.joyent.com/cloudapi/
+[cloudapi]: https://apidocs.tritondatacenter.com/cloudapi/
 [acuguide]: https://docs.joyent.com/jpc/rbac
```
Comment (travispaul) [inline README.md:154]: Maybe this link can be replaced with: https://docs.tritondatacenter.com/public-cloud/rbac

File: README.md (hunk 1 above)
Comment (bahamat) [reply README.md]: Thank you! fixed

File: README.md (hunk 2 above)
Comment (bahamat) [reply README.md]: Also fixed

**Review by travispaul — APPROVED** [review]

**Pull request merged**